DFE-Digital/apply-for-teacher-training
# Tidy up performance dashboards

## Context

Copy and pasting code within dashboards has meant a few bad habits have been repeated. 

## Changes proposed in this pull request

* Remove repeated `headline-stat` `id` attributes (`id`s should be unique)
* Add useful page fragment identifiers
* Use `data-qa` for test hooks
* Ensure tile component always has the same bottom border (which means this doesn’t need to be set in every template)
* Update markup for dashboards (consistent spacing and heading sizes)

## Guidance to review



## Link to Trello card



## Things to check
